feat: First draft for clisk contributor documentation (#282)
cozy/cozy.github.io: src/tutorials/CliSK/5 - Best practices/5.1 - Coding standards.md

A yarn script is provided to lint the code. It will respect most of our coding standards. Simply use the `yarn lint --fix` command
#### Logs

> Regarding the logs you might put in your code, it is recommended to avoid logging sensible information once the konnectors is ready to go to production.

Reviewer: ```suggestion
Regarding the logs you might put in your code, it is recommended to avoid logging sensible information once the konnector is ready to go to production.
```

It might me obvious but better safe than sorry. 

> On the other hand, keep it simple and practical. For debug purpose, it's good to have a starting log at the beginning of each *mains* functions, just to track possible bugs more efficiently. You can use the common type of log such as :

Reviewer: ```suggestion
On the other hand, keep it simple and practical. For debug purpose, it's good to have a starting log at the beginning of each *main* functions, just to track possible bugs more efficiently. You can use the common type of log such as :
```

- `info` , to keep track of the execution steps, state of a page or anything worth seeing during the execution like number of found files for example

> - `debug`, only visible when the instance is in debug mode once in production. Could be use to have a better overview of whats happening during the execution without polluting the log flow.

Reviewer: ```suggestion
- `debug`, only visible when the instance is in debug mode once in production. Could be used to have a better overview of what is happening during the execution without polluting the log flow.
```

> - `error` , to -of course- clearly show an error as occurred. However it is rarely use in production as we will prefer to throw an error with the JS native `throw new Error()` . It will stop the execution of the konnector if a critical error show up like a website maintenance.

Reviewer: ```suggestion
- `error` , to -of course- clearly show an error as occurred. However it is rarely used in production as we will prefer to throw an error with the JS native `throw new Error()` . It will stop the execution of the konnector if a critical error show up like a website maintenance.
```
